jobserver/authorization/admins.py

```diff
@@ -0,0 +1,16 @@
+from django.db import transaction
+from environs import Env
+
+from ..models import User
+from .roles import CoreDeveloper
+
+
+env = Env()
+
+
+@transaction.atomic
```

Comment:
question: I used the previously existing  module as a starting point for this module. 

Can this `transaction.atomic` decorator can be removed, since we now only have a single `.save()`?